### DISCUSSION TITLE
Migration

### DISCUSSION THREAD

**Discussion by Fasapokhai** [dataplat/dbatools#7796] [Q&A]
Hello All. 
I am new to powershell...
I am having some issues with migration.

below is my command

Copy-DbaCredential -Source dysrp -Destination dysrd  -Credential ID\12345 -force -whatif

Result

WARNING: [10:35:21][Copy-Credential] Error creating credential | You cannot call a method on a null-valued expression.
What if: Performing the operation "Set variable" on target "Name: __dbatools_interrupt_function_78Q9VPrM6999g6zo24Qn83m09XF56InEn4hFrA8Fwhu5xJrs6r Value: True".
WARNING: [10:35:21][Copy-Credential] Error creating credential | You cannot call a method on a null-valued expression.

Please advise.
Thanks!

**Comment by dbatoolsuser** [comment]
this looks like permissions issue, make sure you are local admin and run powershell with elevation.

**Reply by potatoqualitee** [reply]
I'm not so sure (sorry, I didnt see any discussions previously but have since enabled notifications) -- those are both remote servers. Elevation is only required when they are local, and an error pops up

![image](https://user-images.githubusercontent.com/8278033/180430650-efb6e984-a099-4827-8c34-68352e378037.png)

I believe this issue was fixed by Andreas earlier this year, please try again and if you get an error, please execute `Get-DbatoolsError` and let us know.
